src/components/wHome.js

```diff
@@ -0,0 +1,45 @@
+// wHome.js
+/* eslint-disable max-len */
+/* eslint-disable no-trailing-spaces */
```

Comment:
### 🛑 Required Change

- [ ] Kindly avoid disabling linters because they guide us in writing code that is clean and well formatted.

Reply:
Done, thank you